### ISSUE TITLE
Resources related to python ,data structures,data analytics,machine learning and artificial intelligence

### ISSUE THREAD

**Issue by nidhi-2619** [nidhi-2619/Resources#1]

**Comment by yamtimor** [comment]
Does it need to be in PDF format or can videos and courses links goes as well?

**Comment by nidhi-2619** [comment]
> Does it need to be in PDF format or can videos and courses links goes as well?
Anything but the content quality should be good and beneficial as well

**Comment by nidhi-2619** [comment]
@yamtimor Yes,  but the content should be quality content, and easily accessible.

**Comment by yamtimor** [comment]
I saw you've already covered most of the main Data Science subjects, so I don't want to repeat. However, there is one video that definitely changed my way of coding in Python and I highly recommend you add it. Instead of PR, I'll just send the link here: https://www.youtube.com/watch?v=7lmCu8wz8ro&ab_channel=CodingTech

**Comment by nidhi-2619** [comment]
@yamtimor Thank You so much I'll surely add it .
